README.md

```diff
@@ -39,33 +39,24 @@ If no matching `routingGroup` found, router will route to `adhoc` group of clust
         "includeInRouter": true,
         "localPort": 8081,
         "name": "presto1",
-        "prefix": "/",
-        "preserveHost": "true",
         "proxyTo": "http://presto1.lyft.com",
-        "routingGroup": "adhoc",
-        "trustAll": "true"
+        "routingGroup": "adhoc"
     },
     {
         "active": true,
         "includeInRouter": true,
         "localPort": 8083,
         "name": "presto3",
-        "prefix": "/",
-        "preserveHost": "true",
         "proxyTo": "http://presto3.lyft.com",
-        "routingGroup": "adhoc",
-        "trustAll": "true"
+        "routingGroup": "adhoc"
     },
     {
         "active": true,
         "includeInRouter": true,
         "localPort": 8082,
         "name": "presto2",
-        "prefix": "/",
-        "preserveHost": "true",
```

Comment:
Whats the intention of these config values ? Why werent they getting used ?

Reply:
These config values are set default by `proxy-server` and we are not overriding those. When we make config calls, they are showing up. I made getters `protected` to avoid printing these.